dump.osm

```diff
@@ -0,0 +1,2934 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Unless this file is being used for tests, I don't know why we need it.

Reply:
This was from a commit that wasn't supposed to be included. Fixed